--- setup.php	Sat Apr 21 03:06:07 2001
+++ setup.php	Fri May 24 11:17:08 2002
@@ -1,3 +1,4 @@
+<?php /* Modified at 1 places by ri_once */ ?>
 <?php
    /*
     *  motd v1.2
@@ -21,7 +22,7 @@
       $motd_file = "../config/motd.php";
 
       if(file_exists($motd_file)) {
-        include($motd_file);
+        /* '_once' Added by ri_once */ include_once($motd_file);
       }
       else {
 	print "<hr width=\"50%\">";
